roles/scaffold_rm_facts/templates/module_utils/network_os/facts/resource/resource.py.j2

```diff
@@ -73,8 +73,10 @@ class {{ resource|capitalize }}Facts(object):
                     objs.append(obj)
         facts = {}
         if objs:
+            facts['{{ resource }}'] = []
```

Comment:
Quick question: Would this break for facts with are inherently not a list of items (e.g., lacp)?

Reply:
By default, the value stored for the fact is a list of dict. In case the fact is not a list item it is converted to list of dict and stored in facts as per the current logic. Adding this line won't change existing behavior.